migration/migrator/data/course_tables.sql

```diff
@@ -41,7 +27,6 @@ CREATE TYPE public.notifications_component AS ENUM (
 
 SET default_tablespace = '';
 
-SET default_with_oids = false;
```

Comment:
why is this removed in this PR?
(I dont know where it's used, I see it was added 2 months ago with PR #7338